pdp/static/js/pdp_controls.js

```diff
@@ -117,6 +117,7 @@ function Colorbar(div_id, layer) {
     this.layer = layer;
     this.minimum = 0.0;
     this.maximum = 0.0;
+    this.midpoint = 0.0;
```

Comment:
So it may have seemed odd to you that min/max are saved as state while the midpoint is not. But the reason that getting the min/max is a relatively expensive operation (an async network query), while the midpoint is simple client-side calculation. I would prefer to keep it that way. Maintain as little state as possible. At the very least, midpoint should be a variable local to the function that is using it (`redraw()`).